google/datalab/contrib/mltoolbox/_archive.py

```diff
@@ -36,6 +37,10 @@ def extract_archive(archive_path, dest):
 
   try:
     tmpfolder = None
+
+    if not (tf.gfile.Exists(archive_path) and not tf.gfile.IsDirectory(archive_path)):
```

Comment:
"if (not tf.gfile.Exists(archive_path)) or (tf.gfile.IsDirectory(archive_path)):" seems easier to understand.